clusters/threat-actor.json

```diff
@@ -644,14 +638,11 @@
           "https://www.secureworks.com/research/threat-profiles/bronze-export",
           "https://www.pwc.co.uk/cyber-security/assets/cyber-threats-2019-retrospect.pdf",
           "https://www.justice.gov/opa/pr/seven-international-cyber-defendants-including-apt41-actors-charged-connection-computer",
-          "https://assets.documentcloud.org/documents/7210602/FLASH-AC-000133-TT-Published.pdf"
+          "https://assets.documentcloud.org/documents/7210602/FLASH-AC-000133-TT-Published.pdf",
+          "https://www.cfr.org/cyber-operations/winnti-umbrella"
         ],
         "synonyms": [
           "Winnti Umbrella",
```

Comment:
```suggestion
          "Grayfly",
```
Some org split APT41 into two arms, one espionage, and the other cybercrime. Grayfly is the espionage arm of APT41.
https://symantec-enterprise-blogs.security.com/blogs/threat-intelligence/apt41-indictments-china-espionage
https://symantec-enterprise-blogs.security.com/blogs/threat-intelligence/grayfly-china-sidewalk-malware

![Grayfly Blackfly](https://user-images.githubusercontent.com/49360849/185649170-e49ff697-8287-4c93-93ff-510c7ee52b38.png)

Reply:
Indeed very good point.

Reply:
Is there a way to include persona names?

Reply:
Do we want to split then? If so, what's the best way to model the relationships between them?

Microsoft also does a split and uses BARIUM (cybercrime) and LEAD (espionage)
https://www.microsoft.com/security/blog/2017/01/25/detecting-threat-actors-in-recent-german-industrial-attacks-with-windows-defender-atp/

Reply:
Yeah right, so does crowdstrike, Symantec and  secureworks.. but FE tracks the whole umbrella as `APT41 and PwC tracks them as `Red Kelpie`. Now, the question how we fit them with two different arms?

Reply:
not sure what is done in other cases like that usually?

Would it make sense to have 3 entities:
 - One for the cybercrime branch
 - One for the espionage branch
 - One for the whole operation

And we would explicit what their relationship is in the descriptions?

Or could we also add a `"type": "part-of"` in the `related` field?
`
        {
          "dest-uuid": "24110866-cb22-4c85-a7d2-0413e126694b", # APT41 uuid
          "tags": [
            "estimative-language:likelihood-probability=\"likely\""
          ],
          "type": "part-of"
        }
`

Reply:
I was thinking about it. `part-of` is indeed a good idea as relationship to describe sub-group. 

Something we could add is the standard meta fields `motive` and start to standardise the value in a taxonomy.

There are some existing motive classification such as a the one from Veris:

![motive](https://user-images.githubusercontent.com/3309/185739473-4db20c60-b23d-4813-8e80-873dca5f1b20.png)

Then this would help to clearly define the sub-group. Let me know what you think about it, I can create a new dedicated taxonomy for each of the meta fields where we would need a vocabulary.

Reply:
Yes, I like the @Mathieu4141's idea. But he already merged the `APT41` subgroups into one, we can break it into subgroups after we find a way to add `motive` field to meta section.

Reply:
Sounds great. For the names attached to a threat-actor, we would need then to create a specific new galaxy for persona and then add relationships between the threat-actor and the persona. I think it's a good idea as we had the exact same discussion about the owners of companies mentioned in `surveillance-vendor`.